pkg/skaffold/schema/profiles.go

```diff
@@ -253,15 +253,15 @@ func applyProfile(config *latest.SkaffoldConfig, profile latest.Profile) error {
 // tryPatch is here to verify patches one by one before we
 // apply them because yamlpatch.Patch is known to panic when a path
 // is not valid.
-func tryPatch(patch yamlpatch.Operation, buf []byte) (valid bool) {
+func tryPatch(patch yamlpatch.Operation, buf []byte) (err error) {
 	defer func() {
 		if errPanic := recover(); errPanic != nil {
-			valid = false
+			err = fmt.Errorf("invalid path: %s", patch.Path)
```

Comment:
To be clear, this is not a new error, this was the pre-existing error. If I'm going to change it, it'll become "yamlpatch panic'd", or I'll just wrap up `errPanic`, since I have no test case for this.

Reply:
ACK. we are trying to collect distinct errors that happen during skaffold run and adding an error code will help us